components/kyma-environment-broker/internal/process/provisioning/event_hub_step_test.go

```diff
@@ -415,20 +426,22 @@ func fixAccountProviderGardenerCredentialsError() hyperscalerautomock.AccountPro
 	return accountProvider
 }
 
-func fixEventHubStep(memoryStorageOp storage.Operations, accountProvider hyperscalerautomock.AccountProvider) *ProvisionAzureEventHubStep {
-	step := NewProvisionAzureEventHubStep(memoryStorageOp,
-		NewFakeHyperscalerProvider(NewFakeNamespaceClientHappyPath()),
-		&accountProvider,
-		context.Background(),
-	)
-	return step
+func fixEventHubStep(memoryStorageOp storage.Operations, hyperscalerProvider azure.HyperscalerProvider,
+	accountProvider hyperscalerautomock.AccountProvider) *ProvisionAzureEventHubStep {
+	return NewProvisionAzureEventHubStep(memoryStorageOp, hyperscalerProvider, &accountProvider, context.Background())
 }
 
 func fixProvisioningOperation(t *testing.T) internal.ProvisioningOperation {
 	op := internal.ProvisioningOperation{
-		Operation: internal.Operation{},
+		Operation: internal.Operation{
+			ID:         fixOperationID,
+			InstanceID: fixInstanceID,
+		},
 		ProvisioningParameters: `{
 			"plan_id": "4deee563-e5ec-4731-b9b1-53b42d855f0c",
+			"ers_context": {
+        		"subaccount_id": "` + fixSubAccountID + `"
```

Comment:
(nit) indention

Reply:
Github is not showing this correctly.

Reply:
problem is mixture of spaces and tabs ;)

Reply:
Rewrote it all.